app/src/features/observations/BlockObservationPage.tsx

```diff
@@ -54,16 +59,20 @@ const BlockObservationPage = () => {
 
   // Ability to bypass showing the 'Are you sure you want to cancel' dialog
   const [enableCancelCheck, setEnableCancelCheck] = useState(true);
-  const [observationWithDetails, setObservationWithDetails] = useState<IBlockObservationForm>(
-    BlockObservationInitialValues
-  );
   const [tableData, setTableData] = useState<any[][]>([[, , , , , , , , , , , , , ,]]);
```

Comment:
Not necessarily something to action now: we'll need to devise all of this table stuff, in general, to handle different headers/schemas/data/etc.  In this case what caught my eye was the hardcoded default that presumably has the right number of cells of the moose, but wouldn't be right for other kinds of data.  Possibly we would need to handle a null default, or have the default based on the headers config, or something along those lines.

Reply:
for sure, good point :) a lot of the stuff is done specifically to handle the moose use case for now 
but a good thing to keep in mind moving forward 💯